dns/dnscrypt-proxy2/src/opnsense/mvc/app/models/OPNsense/Dnscryptproxy/Whitelist.xml

```diff
@@ -0,0 +1,19 @@
+<model>
+    <mount>//OPNsense/dnscryptproxy/whitelist</mount>
+    <description>dnscrypt-proxy Whitelist configuration</description>
+    <version>0.1.0</version>
+    <items>
+        <whitelists>
+            <whitelist type="ArrayField">
+                <enabled type="BooleanField">
+                    <default>1</default>
+                    <Required>Y</Required>
+                </enabled>
+                <name type="TextField">
+                    <default></default>
```

Comment:
no default?

Reply:
Why? Array Fields are usually empty and enabled in most plugins?

Reply:
Yes but then, for a clean solution, you could remove the default tag

Reply:
You are totally right :)